api/get-started/your-first-extension.md

```diff
@@ -14,6 +14,11 @@ In this topic, we'll teach you the fundamental concepts for building extensions.
 ```bash
 npm install -g yo generator-code
 ```
+Allow the terminal to recognize `code` command by modifying the environment variable:
+
+```
+setx Path "%Path%;C:\Users\USERNAME\AppData\Local\Programs\Microsoft VS Code\bin"
```

Comment:
Add `code` to Path environment variable for the VS Code terminal to identify it for all future references